ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -518,7 +520,7 @@ conformance_test(
 )
 
 conformance_test(
-    name = "conformance-test-in-memory-fan-out",
```

Comment:
NOTE: in-memory fan-out is enabled by default

Reply:
maybe then it makes sense to remove `enableInMemoryFanOutForLedgerApi` from `IndexServiceConfig`?

Reply:
WIll do in a later stage so we have a transition period for clients.